Portfolio mobile

Munyabelden opened the pull request:
Added  a popup on see project button click

Favourezeugwa commented on style.css:
```diff
@@ -366,6 +374,161 @@ main {
+.card.active {
+  transform: scale(1);
+}
+
+.card-details {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  z-index: 100;
+  background: #fff;
+  border-radius: 10px;
+  box-shadow: 0 0 0 1000px rgba(193, 199, 208, 0.9);
+  margin: 0 auto;
+  pointer-events: all;
+  max-width: 400px;
+  padding-block: 3%;
+  padding-inline: 4%;
+  text-align: start;
+}
+
```
- I think it will be better if we added some margins to the sides of your card details so that it looks like that of Figma.

Here is what I can see compared to Figma 👇🏽 
⚠️ Kindly ignore the blank image from my Figma screenshot 😉 
|  Your Design  | Figma Design |
| :---:   | :---: | 
| <img width="379" alt="Screenshot 2023-02-08 at 05 00 15" src="https://user-images.githubusercontent.com/99246359/217512622-d5f4312a-f629-4a3d-8d64-904da7a53f52.png"> |!<img width="310" alt="Screenshot 2023-02-08 at 05 08 39" src="https://user-images.githubusercontent.com/99246359/217513241-855eefeb-cc80-4f21-b3de-d35886bd1ad8.png">  |

Favourezeugwa reviewed (changes requested):
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi Team 👋🏽  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![close lady](https://user-images.githubusercontent.com/99246359/207367678-ef4e9b86-75b7-4d8e-b9b5-899f2e87c934.gif)

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Munyabelden commented:
Hi @Favourezeugwa ,

thank you for the review
Im facing two problems when I put my html in js all the images on the page are not working
![Screenshot (19)](https://user-images.githubusercontent.com/116732900/217531098-151283d2-4143-491d-9170-6f2691fef7cc.png)
and no one seems to be able to understand my problem I have tried everything I can 
can u help me please

Munyabelden commented:
Hello I'm not quite sure how to do everything in js can you help me out
please

On Wed, 08 Feb 2023, 13:12 Favour Ezeugwa, ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️
>
> Hi Team 👋🏽 ,
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> [image: close lady]
> <https://user-images.githubusercontent.com/99246359/207367678-ef4e9b86-75b7-4d8e-b9b5-899f2e87c934.gif>
> Highlights 💯
>
>    - No Linter errors ✔️
>    - Descriptive Pull request summary✔️
>    - Good commit messages ✔️
>    - Professional Readme file ✔️
>
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.js
> <https://github.com/Munyabelden/munyabelden.github.io/pull/17#discussion_r1099974573>
> :
>
> > +const popUpCard = document.querySelectorAll('[data-modal-target]');
>
> +const closeCard = document.querySelectorAll('[data-close]');
>
>
>
>    - Great job implementing the popup modal 💪🏽
>    - I believe this project requires us to do it differently, from the
>    way you have done yours.
>    Here is the requirement 👇🏽
>
> [image: Screenshot 2023-02-08 at 04 51 58]
> <https://user-images.githubusercontent.com/99246359/217509932-4413ef50-1932-4155-b91a-f6dd0316f9f4.png>
>
>    - The JavaScript code should be implemented as an array of objects to
>    store JavaScript objects containing the information for each project.
>    - Each object should contain at least the following information about
>    the project: name, description, featured image, technologies, link to live
>    version, and link to the project source code.
>
> Something like this 👇🏽
>
> const projects = [{
>
>     name: "Submarine game",
>
>     description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
>
>     image: "link_to_image.jpg",
>
>     technologies: ["Phaser3", "JavaScript", "Jest"],
>
>     link: "link_to_live_version.com",
>
> },
>
> {
>
>     name: "Submarine game",
>
>     description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
>
>     image: "link_to_image.jpg",
>
>     technologies: ["Phaser3", "JavaScript", "Jest"],
>
>     link: "link_to_live_version.com",
>
> }
>
> ]
>
>
>
>    - Your project card should also be in the js file too. No Project card
>    should be on your HTML file. 🌸
>
> ------------------------------
>
> In style.css
> <https://github.com/Munyabelden/munyabelden.github.io/pull/17#discussion_r1099987048>
> :
>
> > +
>
> +.card-details {
>
> +  position: fixed;
>
> +  display: flex;
>
> +  flex-direction: column;
>
> +  align-items: center;
>
> +  justify-content: center;
>
> +  z-index: 100;
>
> +  background: #fff;
>
> +  border-radius: 10px;
>
> +  box-shadow: 0 0 0 1000px rgba(193, 199, 208, 0.9);
>
> +  margin: 0 auto;
>
> +  pointer-events: all;
>
> +  max-width: 400px;
>
> +  padding-block: 3%;
>
> +  padding-inline: 4%;
>
> +  text-align: start;
>
> +}
>
> +
>
>
>
>    - I think it will be better if we added some margins to the sides of
>    your card details so that it looks like that of Figma.
>
> Here is what I can see compared to Figma 👇🏽
> ⚠️ Kindly ignore the blank image from my Figma screenshot 😉
> Your Design Figma Design
> [image: Screenshot 2023-02-08 at 05 00 15]
> <https://user-images.githubusercontent.com/99246359/217512622-d5f4312a-f629-4a3d-8d64-904da7a53f52.png>
> ![image: Screenshot 2023-02-08 at 05 08 39]
> <https://user-images.githubusercontent.com/99246359/217513241-855eefeb-cc80-4f21-b3de-d35886bd1ad8.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Munyabelden/munyabelden.github.io/pull/17#pullrequestreview-1288934187>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A32THZHH4UVX5GD5B2WXG23WWN5TRANCNFSM6AAAAAAUVBTLBU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

KayLemba commented on index.js:
```diff
@@ -1,25 +1,179 @@
+      html: 'html',
+      css: 'css',
+      javascript: 'javascript',
+      ruby: 'ruby on rails',
+      github: 'github',
+      bootstrap: 'bootstrap'
+    },
+    live: 'https://munyabelden.github.io/',
+    source: 'https://github.com/Munyabelden/munyabelden.github.io'
+  },
+  {
+    id: 'card-2',
+    title: 'Multi-post Stories',
+    list: {
+      title: 'canopy',
+      dev: 'Backend-dev',
+      year: '2015'
+    },
+    images: {
+      large: './images/Portfolio3x.png',
+      small: './images/Portfolio.jpg'
+    },
+    information: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: {
+      html: 'html',
+      css: 'css',
+      javascript: 'javascript',
+      ruby: 'ruby on rails',
+      github: 'github',
+      bootstrap: 'bootstrap'
+    },
+    live: 'https://munyabelden.github.io/',
+    source: 'https://github.com/Munyabelden/munyabelden.github.io'
+  },
+  {
+    id: 'card-1',
+    title: 'Tonic',
+    list: {
+      title: 'canopy',
+      dev: 'Backend-dev',
```
Good job trying to implement this but as suggested by the previous code reviewer your cards should be displayed dynamically using javascript DOM manipulation so all your code in your card section of your index.html file should be deleted and the cards should be rendered dynamically.

I also noticed your pop-up is only working on one card in your desktop version and not working in the last 3 cards and also in your mobile view your card are not popping up please kindly fix this issue for you to proceed to the next project both pop up windows for desktop and mobile view shouls be fully functional

KayLemba reviewed (changes requested):
Hi @Munyabelden    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Munyabelden commented:
Hello how do I create everything in js

On Thu, 09 Feb 2023, 08:32 Kalolo Chola Lemba, ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @Munyabelden <https://github.com/Munyabelden> 👋
> Good job so far working on the project requirements!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> To Highlight 🎉
> No linter errors. ✔️
> GitHub flow is followed. ✔️
>
> Required Changes ♻️
> Check the comments under the review in the inline comments.
>
> Optional suggestions
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the
> approval of this PR. However, I strongly recommend you to take them into
> account as they can make your code better.
>
> You can also consider:
> N/A
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.
>
> As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.
> ------------------------------
>
> In index.js
> <https://github.com/Munyabelden/munyabelden.github.io/pull/17#discussion_r1101029606>
> :
>
> > -
>
>
>
> +const popUpCard = document.querySelectorAll('[data-modal-target]');
>
> +const cardElement = document.querySelector('.card');
>
> +
>
> +const cardArray =  [
>
> +  {
>
> +    id: 'card-1',
>
> +    title: 'Tonic',
>
> +    list: {
>
> +      title: 'canopy',
>
> +      dev: 'Backend-dev',
>
> +      year: '2015'
>
> +    },
>
> +    images: {
>
> +      large: './images/Portfolio3x.png',
>
> +      small: './images/Portfolio.jpg'
>
> +    },
>
> +    information: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
>
> +    technologies: {
>
> +      html: 'html',
>
> +      css: 'css',
>
> +      javascript: 'javascript',
>
> +      ruby: 'ruby on rails',
>
> +      github: 'github',
>
> +      bootstrap: 'bootstrap'
>
> +    },
>
> +    live: 'https://munyabelden.github.io/',
>
> +    source: 'https://github.com/Munyabelden/munyabelden.github.io'
>
> +  },
>
> +  {
>
> +    id: 'card-2',
>
> +    title: 'Multi-post Stories',
>
> +    list: {
>
> +      title: 'canopy',
>
> +      dev: 'Backend-dev',
>
> +      year: '2015'
>
> +    },
>
> +    images: {
>
> +      large: './images/Portfolio3x.png',
>
> +      small: './images/Portfolio.jpg'
>
> +    },
>
> +    information: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
>
> +    technologies: {
>
> +      html: 'html',
>
> +      css: 'css',
>
> +      javascript: 'javascript',
>
> +      ruby: 'ruby on rails',
>
> +      github: 'github',
>
> +      bootstrap: 'bootstrap'
>
> +    },
>
> +    live: 'https://munyabelden.github.io/',
>
> +    source: 'https://github.com/Munyabelden/munyabelden.github.io'
>
> +  },
>
> +  {
>
> +    id: 'card-1',
>
> +    title: 'Tonic',
>
> +    list: {
>
> +      title: 'canopy',
>
> +      dev: 'Backend-dev',
>
>
> Good job trying to implement this but as suggested by the previous code
> reviewer your cards should be displayed dynamically using javascript DOM
> manipulation so all your code in your card section of your index.html file
> should be deleted and the cards should be rendered dynamically.
>
> I also noticed your pop-up is only working on one card in your desktop
> version and not working in the last 3 cards and also in your mobile view
> your card are not popping up please kindly fix this issue for you to
> proceed to the next project both pop up windows for desktop and mobile view
> shouls be fully functional
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Munyabelden/munyabelden.github.io/pull/17#pullrequestreview-1290501810>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A32THZBTUSBAD6QHR22DHQ3WWSFPHANCNFSM6AAAAAAUVBTLBU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

HAMMAS-SALEEM commented on README.md:
```diff
@@ -66,7 +66,7 @@
 
```
![image](https://user-images.githubusercontent.com/93039359/217890917-605fb6a5-c8fe-4dee-904a-ef546f423f1a.png)

- [ ] You are missing ESLint from your linters setup which checks your JavaScript files. It checks those errors which can break your application :thinking:

HAMMAS-SALEEM commented:
This might be the last review you guys have. If you need any assistance you can reach out to me on slack by searching my name **Hammas** in **full-time-students** channel :+1:

iambenkis commented on index.js:
```diff
@@ -1,25 +1,245 @@
+    css: 'css',
+    javascript: 'javascript',
+    ruby: 'ruby on rails',
+    bootstrap: 'bootstrap',
+    github: 'github',
+    linkLive: 'https://github.com/Munyabelden/munyabelden.github.io/',
+    linkSource: 'https://github.com/Munyabelden/munyabelden.github.io',
+  },
+  {
```
- [ ] [**OPTIONAL**] Kindly improve the position of these links :+1: , it will make your page sticks to the Figma design :+1: 
![Screenshot from 2023-02-09 21-52-08](https://user-images.githubusercontent.com/80458199/217923160-54a94244-694b-4b6e-989d-d557a5ee480c.png)

iambenkis reviewed (approved):
Hi @Munyabelden , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![ ](https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif)
## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
